README.md

```diff
@@ -279,11 +284,44 @@ boolean isHTTP = datastore.getOptions().getTransportOptions() instanceof HTTPTra
 #### New Features
 There are new gRPC specific features available to use in this update.
 
-##### Channel Pooling
-To customize the number of channels your client uses, you can update the channel provider in the DatastoreOptions. 
+##### Connection Pool
+A connection pool, also known as a channel pool, is a cache of database connections that are shared and reused to improve connection latency and performance. With this update, now you will be able to configure the channel pool to improve application performance. This section guides you in determining the optimal connection pool size and configuring it within the Java datastore client.
+To customize the number of channels your client uses, you can update the channel provider in the DatastoreOptions.
+###### Determine the best connection pool size
+The default connection pool size is right for most applications, and in most cases there's no need to change it.
+
+However sometimes you may want to change your connection pool size due to high throughput or buffered requests. Ideally, to leave room for traffic fluctuations, a connection pool has about twice the number of connections it takes for maximum saturation. Because a connection can handle a maximum of 100 concurrent requests, between 10 and 50 outstanding requests per connection is optimal. The limit of 100 concurrent streams per gRPC connection is enforced in Google's middleware layer, and you are not able to reconfigure this number.
+
+The following steps help you calculate the optimal number of connections in your channel pool using client-side metrics such as those available from [OpenCensus](https://opencensus.io/guides/grpc/java/#examining-metrics).
```

Comment:
I don't think we have opencensus metrics implemented. Should refrain from suggesting this.